lib/bowling.rb

```diff
@@ -0,0 +1,123 @@
+class Bowling
+  attr_reader :round_score, :total_score, :round, :ball, :strike_bonus, :spare_bonus, :bonus_ball, :bonus_score, :last_strike
+
+  def initialize
+    @round_score = 0
+    @total_score = 0
+    @round = 1 
+    @ball = 1
+    @strike_bonus = nil
+    @spare_bonus = nil
+    @bonus_ball = 1
+    @bonus_score = 0
+    @last_strike = nil
+  end
+
+  def score_input
+    input = $stdin.gets
+    scorer(input.to_i)
+  end
+
+  def input_score_message
+    "Please enter your score for round #{round}, ball #{ball}"
```

Comment:
👍🏻  I like this input prompt - it helps the user keep track of which frame/roll they are entering.